desktop/apps/page/test/end_to_end.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-env mocha */
+import { setup, teardown } from '../../../../test/helpers/end_to_end'
```

Comment:
[Naming things](https://martinfowler.com/bliki/TwoHardThings.html)... "end to end" seems like a more common term, but curious if anyone has a preference on "end to end" vs. "acceptance" vs. "integration".